File: problem 16/Option 01/BoxTest.java
class Box{

    private float length;
    private float width;
    private float height;

    public Box(float h, float w, float l) {
        height = h;
        width = w;
        length = l;
    }

    public float calculateVolume() {
        return (height*width*length);
    }
}

public class BoxTest {
    public static void main(String[] args) {
        Box pcBox = new Box(7, 8, 3); 
        System.out.println("The volume of a box is : " + pcBox.calculateVolume());
    }
}

// on cmd navigated DIR just write " javac BoxTest.java && java BoxTest " if you have JDK run app on your machine

File: problem 16/Option 01/BoxTest.cpp
#include <iostream>

class Box{
    private:
        float length;
        float height;
        float width;

    public:
        Box(float h, float w, float l){
            length = l;
            height = h;
            width = w;
        }
        float calculateVolume(){
            return (width * height * length);
        }
};

int main(){
    Box mid = Box(7, 8, 3);
    std::cout << "The volume of a box is : " << mid.calculateVolume() << std::endl;
    return 0;
}

// on cmd navigated DIR just write " g++ BoxTest.cpp -o BoxTest && BoxTest " if you have DevC++/CodeBlocks run app on your machine